Sources/TypeDecoder/OrderedDictionary.swift

```diff
@@ -51,6 +51,17 @@ public struct OrderedDictionary<K: Hashable, V> {
         }
     }
 
+    /// subscript: get the value at the numeric index specified.
+    public subscript(index: Int) -> V? {
+        get {
+            if 0 ... self.keys.count - 1 ~= index {
```

Comment:
Can we use `keys.indices.contains(index)` instead?

Reply:
done